README.md

```diff
@@ -1,4 +1,193 @@
-# pravega-benchmark
-Performance benchmark tool and scripts for Pravega
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+# Pravega Benchmark Tool
+
+The pravega benchmark tool used for the performance benchmarking of pravega streaming storage cluster.
+This tool performs the throughput and latency analysis for the multi producers and consumers of pravega.
+it also validates the end to end latency. The write and/or read latencies can be stored in the CSV file for later analysis.
+At the end of the performance benchmarking, this tool outputs the 50th, 75th, 95th , 99th and 99.9th latency percentiles.
+
+
+### Prerequisites
+
+- Pravega client : https://github.com/pravega/pravega
+- Java 8+
+
+### Building
+
+Checkout the source code:
+
+```
+git clone https://github.com/pravega/pravega-benchmark
+cd pravega-benchmark
+```
+
+check the pravega client version and add to build.gradle file
+
+```
+For example:
+you may have pravega client version 0.5.0 in this folder:
+
+ls /<home directory>/.m2/repository/io/pravega/pravega-client/
+0.5.0-SNAPSHOT
+
+add this version : 0.5.0-2064.d8dd7cb-SNAPSHOT  to <>/pravega-benchmark/build.gradle file as follows:
+dependencies {
+
+compile "io.pravega:pravega-client:0.5.0-SNAPSHOT",
+"io.pravega:pravega-common:0.5.0-SNAPSHOT",
+....
+....
+}
+```
+
+Build the pravega benchmark Tool:
+
+```
+gradle distTar
+```
+
+untar the pravega benchmark tool to local folder
+
+```
+tar -xvf ./build/distributions/pravega-benchmark.tar -C ./run
+```
+
+Running pravega bencmark tool locally:
+
+```
+kmg@INENMUNEGKL1C:~/kmg-pravega/pravega-benchmark$ ./run/pravega-benchmark/bin/pravega-benchmark  --help
```

Comment:
yes it done.
Since there were too many commits ; I have cleanup and raised seperate PR :
https://github.com/pravega/pravega-benchmark/pull/36

All you review comments are addressed in the above PR.